target-app/Info.plist

```diff
@@ -4,5 +4,11 @@
 <dict>
 	<key>Base URL</key>
 	<string>https://target-api-induction-v2.herokuapp.com/api/v1</string>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string>Used to present offers based on your location</string>
```

Comment:
is this true to the application? we don't really offer anything to the user. 
The value set for  `NSLocationWhenInUseUsageDescription` seems more appropriate.

Reply:
Its true that this string is not precise. Fixed it!